Issue482 mission statement

wd15 opened the pull request:
Add improved version of Olle's mission statement text. The text

```
       <p>
          The {{ site.title }} is a resource and forum for phase field
          practitioners ranging from novices to seasoned code
          developers. The contents of the {{ site.title }} is
          developed through community involvement with a special
          emphasis on providing links and information about codes and
          benchmark problems for phase field modeling. The primary
          goals are to host resources that
          <ul>
            <li>
              provide a set of benchmark problems that facilitate
              comparing and contrasting phase field codes as well as
              provide pedagogical examples for teaching and learning
              phase field methods,
            </li>
            <li>
              increase the adoption of phase field methods in
              engineering and scientific applications and aid in the
              development of quantitatively predictive phase field
              modeling,
            </li>
            <li>
              and, that foster an engaged and integrated phase field
              community with special emphasis on providing networking
              opportunities for graduate students and post-docs.
            </li>
          </ul>
        </p>
```

drjamesawarren commented:
This looks fine.   The "that and, that" in the last bullet is a bit odd IMO, but is perhaps grammatically correct?

wd15 commented:
@drjamesawarren, thanks, fixed text and fixed broken HTML

New text:

      <div class="flow-text mission-statement">
        <p>
          The {{ site.title }} is a resource and forum for phase field
          practitioners ranging from novices to seasoned code
          developers. The contents of the {{ site.title }} is
          developed through community involvement with a special
          emphasis on providing links and information about codes and
          benchmark problems for phase field modeling. The primary
          goals are to host resources that
        </p>
        <ul>
          <li>
            provide a set of benchmark problems that facilitate
            comparing and contrasting phase field codes as well as
            provide pedagogical examples for teaching and learning
            phase field methods,
          </li>
          <li>
            increase the adoption of phase field methods in
            engineering and scientific applications and aid in the
            development of quantitatively predictive phase field
            modeling,
          </li>
          <li>
            and, foster an engaged and integrated phase field
            community with special emphasis on providing networking
            opportunities for graduate students and post-docs.
          </li>
        </ul>
      </div>

tkphd commented:
Renders as:

 > The CHiMaD Phase Field is a resource and forum for phase field practitioners ranging from novices to seasoned code developers. The contents of the CHiMaD Phase Field is developed through community involvement with a special emphasis on providing links and information about codes and benchmark problems for phase field modeling. The primary goals are to host resources that
>
> - provide a set of benchmark problems that facilitate comparing and contrasting phase field codes as well as provide pedagogical examples for teaching and learning phase field methods,
> - increase the adoption of phase field methods in engineering and scientific applications and aid in the development of quantitatively predictive phase field modeling,
> - and, foster an engaged and integrated phase field community with special emphasis on providing networking opportunities for graduate students and post-docs.

tkphd commented:
@heinono1 and @drjamesawarren:

The text strikes me as a good start, but it does not delineate the mission of the CHiMaD Phase Field Working Group and the separate, subsidiary mission of the chimad-phase-field website.  The first sentence and last two bullets seem Working Group related, while the rest of the paragraph and first bullet are Website related. Grammatically it's OK, and a visitor would get some idea of our goals from reading it, but the interleaved messages betray some uncertainty in the soul of this effort. 

`{{ site.title }}` (whatever settles out of the legal analysis of `pfhub`) is not just a website, not just the working group, and not just the community brought together through this [website](https://pages.nist.gov/chimad-phase-field/), [gitter](https://gitter.im/usnistgov/chimad-phase-field), and *ad hoc* networking in the physical realm. It's all those things, and an ideal mission statement would capture that breadth along with the members' commitment to go beyond the meetings and outside the limits of the benchmarks to help the PF community grow.

My opinion is that we should treat this as a "Welcome, here's our scope" rather than a "Mission" statement. There are at least two pathways forward:
1. Cut `and forum` from the first sentence and cut the last bullet point entirely. This will focus on the "mission" of the website as it stands.
2. Generalize the statement to encompass all activities, and spell out each aspect of the effort (workshops, website, online chats, benchmark hackathons, conference tutorials). Parts that are not accessible from the website would have to be incorporated somehow -- e.g., a calendar of workshop events and related presentations at conferences, or a list of resources to help learn phase-field.

tkphd commented:
Consider, instead, minor variations on:
> Welcome to our phase-field community! If you're a beginner, you'll find resources and a crash course on the technique. If you're an expert, you'll find tools to verify the quality of your simulations. If you're anywhere in-between, you'll find a community built around shared interest in phase-field and a commitment to opening up our research for the benefit of all.
